### PR TITLE
Makes midround antagonists not spawn if the shuttle is past the point of no return

### LINKED ISSUE #64756: midround ghost antags can roll during shuttle departure

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

I was considering doing this for latejoin too, but the issue report was solely for midround, so that's all I'm doing. I can easily add it in if it's wanted, but I thought it would ruin special events like [insert video of a guy latejoining traitor, rushing chem bombs and bombing evac (i cant find it)].

## Why It's Good For The Game

Blob/Blob infection can roll while the shuttle is docked/already gone, which I don't find necessarily any fun for anyone.
Closes https://github.com/tgstation/tgstation/issues/64756

## Changelog

:cl:
fix: Midround antagonists cannot be rolled while the shuttle is docked at the station, or already gone to Central Command.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm [hunk 1]
```diff
@@ -82,6 +82,8 @@
 		var/threat = round(mode.threat_level/10)
 		if (job_check < required_enemies[threat])
 			return FALSE
+		if(EMERGENCY_PAST_POINT_OF_NO_RETURN)
```
Comment (Mothblocks) [inline code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm:85]: It shouldn't get to this point. This should be placed in the code that actually handles the midround rolls.

Comment (JohnFulpWillard) [reply code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm]: You're right, sorry.

**Comment by Redtrogdor** [comment]
Pretty sure this was the video you're thinking of: https://www.youtube.com/watch?v=kAokf6d1dLA

**Comment by jlsnow301** [comment]
Thank god

**Comment by JohnFulpWillard** [comment]
> 

yes it was thank you

**Comment by Mothblocks** [comment]
The comment is useless but whatever

**Pull request merged**